### PR TITLE
[en]added explanation for unprivileged access

### PR DESCRIPTION
**Pull request by Saliha067**
Part of: https://github.com/kubernetes/website/issues/38436

The preview page:
- [Link](https://deploy-preview-39647--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/#secret)

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -934,7 +934,7 @@ A container using a Secret as a [`subPath`](#using-subpath) volume mount will no
 receive Secret updates.
 {{< /note >}}
 
-For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
+As a default, Secrets are readable only by `root`. For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
```
Comment (tengqm) [inline content/en/docs/concepts/storage/volumes.md:937]: What is this?

Comment (Saliha067) [reply content/en/docs/concepts/storage/volumes.md:937]: @tengqm Part of this: https://github.com/kubernetes/website/issues/38436

Comment (sftim) [reply content/en/docs/concepts/storage/volumes.md:937]: We should definitely check that the assertion is accurate.

Comment (Saliha067) [reply content/en/docs/concepts/storage/volumes.md:937]: @sftim is there anything we have add more on this?

**Comment by kbhawkey** [comment]
👋 @Saliha067 .
I am closing this pull request. If you have questions, feel free to reopen it.
/close

**Pull request closed without merging**